{{cookiecutter.project_slug}}/templates/demo_styles.html

```diff
@@ -1 +1,352 @@
 {% extends 'base.html' %}
+
+{% block title %}
+    Demo Styles - {{ block.super }}
+{% endblock title %}
+
+{% block main %}
+  <div class="container">
+
+		<main role="main">
```

Comment:
I see tabs here!

Reply:
there's some wizardry going on here, as these tabs aren't actually in the code, they're just showing up here?

Reply:
Nope, they were definitely there.

I've removed them.

Reply:
that's very curious, i double checked this. Not the first time i've seen it happen though 🧐 thanks anyway